backend/geonature/core/imports/routes/__init__.py

```diff
@@ -5,14 +5,38 @@
 from geonature.core.imports.schemas import DestinationSchema
 from geonature.core.imports.blueprint import blueprint
 from geonature.utils.env import db
+from flask_login import current_user
 
 
 @blueprint.route("/destinations/", methods=["GET"])
-@login_required
-def list_destinations():
+def list_all_destinations():
+    """
+    Return the list of all destinations.
+
+    Returns:
+    -------
+    destinations : List of Destination
+        List of all destinations.
+    """
+
     schema = DestinationSchema()
     destinations = Destination.query.all()
-    # FIXME: filter with C permissions?
+    return schema.dump(destinations, many=True)
+
+
+@blueprint.route("/destinations/allowed", methods=["GET"])
+@login_required
+def allowed_destinations():
+    """
+    Return a list of allowed destinations for the current user.
+
+    Returns:
+    -------
+    destinations : List of Destination
+        List of allowed destinations for the current user.
+    """
+    schema = DestinationSchema()
+    destinations = Destination.allowed_destinations(current_user)
```

Comment:
```suggestion
    destinations = Destination.allowed_destinations()
```
ou alors `g.current_user`